openroad package fails to build

https://github.com/hdl/conda-eda/actions/runs/3492792934/jobs/5846941441

```
  CMake Error at src/mpl2/CMakeLists.txt:42 (find_package):
    By not providing "Findortools.cmake" in CMAKE_MODULE_PATH this project has
    asked CMake to find a package configuration file provided by "ortools", but
    CMake did not find one.
  
    Could not find a package configuration file provided by "ortools" with any
    of the following names:
  
      ortoolsConfig.cmake
      ortools-config.cmake
  
    Add the installation prefix of "ortools" to CMAKE_PREFIX_PATH or set
    "ortools_DIR" to a directory containing one of the above files.  If
    "ortools" provides a separate development package or SDK, be sure it has
    been installed.
  ```
seems like a new? dependency is missing.

--- a/pnr/openroad/disable-mpl2.patch
+++ b/pnr/openroad/disable-mpl2.patch
@@ -1,0 +1,56 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 99752d275..74a1ef540 100755
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -244,7 +244,6 @@ add_subdirectory(cts)
+ add_subdirectory(grt)
+ add_subdirectory(tap)
+ add_subdirectory(mpl)
+-add_subdirectory(mpl2)
+ add_subdirectory(rcx)
+ add_subdirectory(psm)
+ add_subdirectory(par)
+@@ -309,7 +308,6 @@ target_link_libraries(openroad
+   drt
+   dst
+   mpl
+-  mpl2
+   psm
+   ant
+   par
+diff --git a/src/OpenRoad.cc b/src/OpenRoad.cc
+index dcf1ae707..0f0341ddb 100644
+--- a/src/OpenRoad.cc
++++ b/src/OpenRoad.cc
+@@ -57,7 +57,6 @@
+ #include "gui/MakeGui.h"
+ #include "ifp//MakeInitFloorplan.hh"
+ #include "mpl/MakeMacroPlacer.h"
+-#include "mpl2/MakeMacroPlacer.h"
+ #include "odb/cdl.h"
+ #include "odb/db.h"
+ #include "odb/defin.h"
+@@ -160,7 +159,6 @@ OpenRoad::~OpenRoad()
+   deleteTritonCts(tritonCts_);
+   deleteTapcell(tapcell_);
+   deleteMacroPlacer(macro_placer_);
+-  deleteMacroPlacer2(macro_placer2_);
+   deleteOpenRCX(extractor_);
+   deleteTritonRoute(detailed_router_);
+   deleteReplace(replace_);
+@@ -213,7 +211,6 @@ void OpenRoad::init(Tcl_Interp* tcl_interp)
+   tritonCts_ = makeTritonCts();
+   tapcell_ = makeTapcell();
+   macro_placer_ = makeMacroPlacer();
+-  macro_placer2_ = makeMacroPlacer2();
+   extractor_ = makeOpenRCX();
+   detailed_router_ = makeTritonRoute();
+   replace_ = makeReplace();
+@@ -247,7 +244,6 @@ void OpenRoad::init(Tcl_Interp* tcl_interp)
+   initTritonCts(this);
+   initTapcell(this);
+   initMacroPlacer(this);
+-  initMacroPlacer2(this);
+   initOpenRCX(this);
+   initPad(this);
+   initRestructure(this);

--- a/pnr/openroad/meta.yaml
+++ b/pnr/openroad/meta.yaml
@@ -13,6 +13,7 @@ source:
       - odb-drt-disable-tests.patch
       - sta-add-install-options.patch
       - gui-find-qt-only-if-enabled.patch
+      - disable-mpl2.patch # until or-tools dep is added
   - url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ lemon_version }}.tar.gz
     md5: e89f887559113b68657eca67cf3329b5
     folder: third_party/lemon

--- a/pnr/openroad/meta.yaml
+++ b/pnr/openroad/meta.yaml
@@ -10,9 +10,9 @@ source:
   - git_url: https://github.com/The-OpenROAD-Project/OpenROAD.git
     git_rev: master
     patches:
-      - odb-drt-disable-tests.patch
-      - sta-add-install-options.patch
-      - gui-find-qt-only-if-enabled.patch
+      - odb-drt-disable-tests.patch # until tests are added to the recope
+      - sta-add-install-options.patch # to enable sta binary distribution
+      - gui-find-qt-only-if-enabled.patch # until gui variant is added
       - disable-mpl2.patch # until or-tools dep is added
   - url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ lemon_version }}.tar.gz
     md5: {{ lemon_md5 }}

--- a/pnr/openroad/meta.yaml
+++ b/pnr/openroad/meta.yaml
@@ -15,7 +15,7 @@ source:
       - gui-find-qt-only-if-enabled.patch
       - disable-mpl2.patch # until or-tools dep is added
   - url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ lemon_version }}.tar.gz
-    md5: e89f887559113b68657eca67cf3329b5
+    md5: {{ lemon_md5 }}
     folder: third_party/lemon
 build:
   # number: 202202031935

